src/index.ts

```diff
@@ -144,6 +153,21 @@ app.delete("/groups/:id", async (req, res) => {
     res.status(200).json();
   }
 });
+// Login method
+app.post("/login", validateUseLogin, async (req, res) => {
+  const { login, password } = req.body;
+
+  const user = await UserModel.getUserByLoginAndPassword(login, password);
```

Comment:
storing user passwords in plain text is a big security risk. please store hashed password